libp2p/Cargo.toml

```diff
@@ -117,11 +111,8 @@ pin-project = "1.0.0"
 libp2p-deflate = { version = "0.39.0", path = "../transports/deflate", optional = true }
 libp2p-dns = { version = "0.39.0", path = "../transports/dns", optional = true }
 libp2p-mdns = { version = "0.43.0", path = "../protocols/mdns", optional = true }
-libp2p-quic = { version = "=0.7.0-alpha.2", path = "../transports/quic", optional = true }
 libp2p-tcp = { version = "0.39.0", path = "../transports/tcp", optional = true }
-libp2p-tls = { version = "=0.1.0-alpha.2", path = "../transports/tls", optional = true }
```

Comment:
Let's keep `libp2p-tls` now that it is soon no longer an alpha.

https://github.com/libp2p/rust-libp2p/pull/3581/

Reply:
Fixed.